src/Sylius/Bundle/AdminBundle/Resources/config/services/twig/component/product.xml

```diff
@@ -29,7 +29,10 @@
                 <argument type="service" id="ux.live_component.live_responder"/>
             </call>
 
-            <tag name="sylius.live_component.admin" key="sylius_admin:product:form" template="@SyliusAdmin/product/form.html.twig" />
+            <tag
+                name="sylius.live_component.admin"
+                key="sylius_admin:product:form"
+            />
```

Comment:
IMO it can be in one line, the same for other tags

Reply:
Fixed everywhere :)